ChangeLog.d/fix-legacy-compression-issue.txt

```diff
@@ -0,0 +1,7 @@
+Bugfix
+   * Fix an issue where TLS 1.2 clients who send a ClientHello message with
+     legacy_compression_methods get a failure in connection because TLS 1.3
+     is enabled by default and the server rejects the ClientHello packet as
+     malformed for TLS 1.3 in a way that stops the fallback to TLS 1.2.
+     fixes #8995, #9243.
```

Comment:
(minor) should be `Fixes`.

(not sure what others think) That's a very long sentence! ChangeLog entries are normally more succinct.

But I don't know if we want to carry on polishing this at this point...

Reply:
We normally do a pass of polishing changelog entries in the release. The changelog entry needs to be technically accurate but we can do editorial corrections like capitalization at that point.